sandbox/apparmor/apparmor.go

```diff
@@ -437,3 +503,9 @@ func MockFeatures(kernelFeatures []string, kernelError error, parserFeatures []s
 	}
 
 }
+
+func MockSnapdAppArmorSupportsReexec(new func() bool) (restore func()) {
```

Comment:
this doesn't seem used outside of this package so it can live in export_test.go where testutil is fine

Reply:
Yep moved in 76a8d13428